testing/test_multicall.py

```diff
@@ -197,3 +197,40 @@ def m2():
     with pytest.raises(exc):
         MC([m2, m1], {}).execute()
     assert l == ["m1 init", "m1 finish"]
+
+
+@hookimpl(hookwrapper=True)
+def m1(arg1, arg2, arg3):
+    yield
+
+
+@hookimpl
+def m2(arg1, arg2, arg3):
+    return arg1, arg2, arg3
+
+
+@hookimpl(hookwrapper=True)
+def w1(arg1, arg2, arg3):
+    yield
+
+
+@hookimpl(hookwrapper=True)
+def w2(arg1, arg2, arg3):
+    yield
+
+
+def inner_exec(methods):
+    return MC(methods, {'arg1': 1, 'arg2': 2, 'arg3': 3}).execute()
+
+@pytest.mark.benchmark
+def test_hookimpls_speed(benchmark):
+    benchmark(inner_exec, [m1, m2])
+
+
+@pytest.mark.benchmark
+def test_hookwrapper_speed(benchmark):
+    benchmark(inner_exec, [w1, w2])
```

Comment:
@ionelmc the wrapper overhead is actually more true to call usage by a `_HookCaller` which internally calls its [`_HookCaller._hookexec()`](https://github.com/pytest-dev/pluggy/blob/benchmarking/pluggy.py#L714) which in turn eventually invokes the [`PlugginManager._inner_hookexec()`](https://github.com/pytest-dev/pluggy/blob/benchmarking/pluggy.py#L277).

My next move once this stuff lands is to add benchmarks around those higher level components such that we speed test the full API code paths.

@ionelmc by the way did you see our question about how to better plubish the benchmarking results for CI or on github? I noticed in your docs that you've got notes on how to render results to json but is there any known way to present results here?